internal/server/task.go

```diff
@@ -89,6 +90,14 @@ func Task(client TaskClient, tmpl template.Template) Handler {
 			assignTo := postFormString(r, "assignTo")
 
 			switch assignTo {
+			case "me":
+			    user, err := client.GetUserDetails(ctx)
+			    if (err != nil){
+			        return err
+			    } else {
+                    task.AssigneeID = user.ID
+                    data.AssigneeUserName = user.DisplayName
+                }
```

Comment:
This is a minor thing, but you seem to have a mixture spaces and tabs on this lines. In `.go` files we just use tabs, so it would be good to stick with that.

This can be a pain to spot, but there should be a setting in your IDE to show whitespace, which will highlight where tabs and spaces are.